libutils/logging.c

```diff
@@ -216,6 +218,11 @@ static void LogToConsole(const char *msg, LogLevel level, bool color)
 
     fprintf(stdout, "%8s: %s\n", LogLevelToString(level), msg);
 
+    if (level == LOG_LEVEL_DEBUG)
+    {
+        fflush(stdout);
+    }
```

Comment:
@vpodzime do you think I should include an fsync() call here just to make more certain the logs are written every time Log() is called? (from an older PR which I have now closed)

Reply:
Yeah, I don't think it is a big deal if it only happens with `LOG_LEVEL_DEBUG`.